railgun/pkg/config/config.go

```diff
@@ -55,6 +57,9 @@ type Config struct {
 	FilterTags           []string
 	WatchNamespace       string
 
+	PublishService string
+	KongAdminAPI   string
```

Comment:
nit: elsewhere in this code we have comments that at least group together related configurations. Please add a comment here that provides some general explanation of why these flags exist and why they're grouped together.